### PR TITLE
Added Web Bluetooth Sample

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This "Read BLE Battery" sample will go with the upcoming "web/updates" article on the Web Bluetooth API.

Note: You can easily play with this on Chrome OS Dev Channel at https://beaufortfrancois.github.io/samples/web-bluetooth

@scheib Can you review this simple code?

### CODE REVIEW

File: web-bluetooth/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+chrome_version: 45
+feature_id: 5264933985976320
+---
+
+<p>The <a href="https://github.com/WebBluetoothCG/web-bluetooth">Web Bluetooth API</a> aims to discover and communicate with devices over the Bluetooth 4 wireless standard using the Generic Attribute Profile (GATT). It is currently only partially implemented in Chrome OS behind the experimental flag <code>chrome://flags/#enable-web-bluetooth</code>.</p>
+
+<p>This sample illustrates the use of the Web Bluetooth API to retrieve battery
+information from a nearby Bluetooth Device advertising Battery information with
+Bluetooth Low Energy. You may want to try this demo with the BLE Peripheral
+Simulator App from the <a href="_blank"
```
Comment (scheib) [inline web-bluetooth/index.html]: Fix up BLE SIM app link; href <-> target.

File: web-bluetooth/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{% capture js %}
+function onButtonClick() {
+  'use strict';
+
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice({filters:[{services:['battery_service']}]})
+  .then(device => {
+    log('> Found ' + device.name);
+    log('Connecting to GATT Server...');
+    return device.connectGATT()
```
Comment (scheib) [inline web-bluetooth/index.html]: add ';'

File: web-bluetooth/index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  })
+  .then(server => {
+    log('Getting Battery Service...');
+    return server.getPrimaryService('battery_service')
```
Comment (scheib) [inline web-bluetooth/index.html]: ;

File: web-bluetooth/index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  })
+  .then(service => {
+    log('Getting Battery Level Characteristic...');
+    return service.getCharacteristic('battery_level')
+  })
+  .then(characteristic => {
+     log('Reading Battery Level...');
+     return characteristic.readValue()
```
Comment (scheib) [inline web-bluetooth/index.html]: ;

File: web-bluetooth/index.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    log('> Found ' + device.name);
+    log('Connecting to GATT Server...');
+    return device.connectGATT()
+  })
+  .then(server => {
+    log('Getting Battery Service...');
+    return server.getPrimaryService('battery_service')
+  })
+  .then(service => {
+    log('Getting Battery Level Characteristic...');
```
Comment (scheib) [inline web-bluetooth/index.html:38]: check for service == null; Sometimes the service won't be found. For code simplicity you could just throw "didn't find the service" or some such.

Comment (scheib) [inline web-bluetooth/index.html:38]: https://github.com/WebBluetoothCG/web-bluetooth/issues/124 if you don't like current spec behavior.

File: web-bluetooth/index.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  });
+};
+{% endcapture %}
+{% include js_snippet.html js=js title='Javascript Snippet' %}
+
+<script>
+document.querySelector('button').addEventListener('click', function() {
+  document.querySelector('#log').textContent = '';
+  if (!navigator.bluetooth) {
+    log('Web Bluetooth API is not available.');
```
Comment (scheib) [inline web-bluetooth/index.html:61]: Hint to use a supported platform + flag?

File: web-bluetooth/index.html (hunk 5 above)
Comment (scheib) [inline web-bluetooth/index.html:38]: Some ChromeOS devices are currently experiencing this.

**Comment by scheib** [comment]
LGTM with some small changes requested.

File: web-bluetooth/index.html (hunk 1 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html]: Done.

File: web-bluetooth/index.html (hunk 2 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html]: Done.

File: web-bluetooth/index.html (hunk 3 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html]: Done.

File: web-bluetooth/index.html (hunk 4 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html]: Done.

File: web-bluetooth/index.html (hunk 5 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html:38]: Added check for service.

File: web-bluetooth/index.html (hunk 6 above)
Comment (beaufortfrancois) [inline web-bluetooth/index.html:61]: Done.

**Comment by beaufortfrancois** [comment]
Thank you @scheib! All nits have been addressed.

@jeffposnick Can you have a quick look and let me know if you're good to merge this?

**Comment by jeffposnick** [comment]
:+1: Thanks for putting this together, @beaufortfrancois!

**Comment by beaufortfrancois** [comment]
https://googlechrome.github.io/samples/web-bluetooth/ is now live ;)

**Pull request merged**